browser/src/slideshow/engine/animationnodes/AnimationBaseNode.ts

```diff
@@ -81,12 +81,14 @@ abstract class AnimationBaseNode extends BaseNode {
 							slideHash,
 							slideWidth,
 							slideHeight,
+							this.aNodeContext.metaSlide._slideShowHandler.getPresenter()._slideRenderer
 						)
```

Comment:
you can access `_SlideShowHandler` through `this.aNodeContext.aContext.aSlideShowHandler`

Reply:
Thanks, I will do the changes